.github/actions/setup-fluvio/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Setup Fluvio
+description: "Download and install Fluvio CLI via FVM"
+inputs:
+  version:
+    description: "Fluvio version to install"
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check Platform Compatibility
+      if: runner.os != 'Linux' && runner.os != 'macOS'
+      shell: bash
+      run: |
+        echo "Current platform $env:RUNNER_OS, is not supported by Fluvio CLI Installer."
```

Comment:
I suggest making this a warning instead of a failing step.

As a warning, always report success for the step and let the install fail in the next step.  This way the log gets the warning to help diagnosis, but we don't end up with the step unintentionally blocking if a different runner.os string comes along.

Reply:
Sounds good!

Reply:
Please check this for details: https://github.com/infinyon/fluvio/pull/3775/commits/52a56563a403875de684c393376b9fd737c40847

Reply:
I think to make it successful, the run script needs to end in `exit 0` or `true`

Reply:
nvm I see it just echos